### DISCUSSION TITLE
Is there any way to get a response code with exec?

### DISCUSSION THREAD

**Discussion by cody-malcolm** [kubernetes-client/csharp#905] [Q&A] [answered]
If we want to check if an exec command executed successfully, is there any way to check the response code, or do we have to determine it based on the stream/buffer we get from the demuxer? For example could we get equivalent codes to the code we'd get from executing `echo $?` after another command in a terminal?

**Comment by tg123** [comment, marked as answer]
there is a channel called `error` (3)

**Reply by tg123** [reply]
https://github.com/kubernetes-client/python/issues/812
